### PR TITLE
fix(monitoring): update grafana dashboard

### PR DESCRIPTION
**Pull request by lenaschoenburg**
Fixes three issues I've spotted:
1. Startup metrics were shown as milliseconds where it should be seconds.
2. CPU usage query was not working with pod filtering
3. Average process instance execution time was showing nonsensical average values.

After the fixes, I've also re-exported the dashboard which updates various versions and the JSON schema.

I think it makes sense to review the first three fix commits and accept the last commit as is or just with a quick glance if anything stands out.

### CODE REVIEW

**Review by deepthidevaki — COMMENTED** [review]
I think the unit should stay "ms". Please see my comment.

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -11086,7 +11086,7 @@
                   }
                 ]
               },
-              "unit": "ms"
+              "unit": "s"
```
[the remark this answers is not in the document]
Comment (oleschoenburg) [reply monitor/grafana/zeebe.json:10648]: This unit declaration belongs to the "Streamprocessor Recovery Time" panel which is based on `zeebe_stream_processor_startup_recovery_time`.

This metric is used as a timer here: https://github.com/camunda/zeebe/blob/6dac668956e2a7a79df9df31bf0f8238e87eb7c3/stream-platform/src/main/java/io/camunda/zeebe/stream/impl/metrics/StreamProcessorMetrics.java#L106 and it reports seconds, not milliseconds.

[the remark this answers is not in the document]
Comment (deepthidevaki) [reply monitor/grafana/zeebe.json:10648]: Ah.. ok. :smile:

**Review by deepthidevaki — APPROVED** [review]

File: monitor/grafana/zeebe.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (oleschoenburg) [reply monitor/grafana/zeebe.json:10648]: You are right about `observeStartupDuration` using milliseconds but that is a different panel where we do use the right unit: https://github.com/camunda/zeebe/blob/c85d613e0b71359f92a264cce3c2fc6239d4498d/monitor/grafana/zeebe.json#L10567

[the remark this answers is not in the document]
Comment (deepthidevaki) [reply monitor/grafana/zeebe.json:10648]: Yes. Understood. In the PR description you mentioned "Startup metrics". So I assumed it is the raft startup time without checking the diff :D

**Pull request merged**